make-money/ad-overview.md

```diff
@@ -0,0 +1,22 @@
+# Ad Units
+
+## Interstitials
+
+
+Interstitials are full screen ads that are displayed in your application. Tap for Tap's interstials are MRAID and VAST compliant, with smart pre-caching for no lag time. The interstital sizes are 640x960 (Portrait) and 960x640 (landscape).
+
+### Moments Based Interstitials
+
+It is our goal to provide the most contextually relevant ads that fit in seamlessly with your application. This increases monetization performance,  while enhancing the user experience. Moments Based Monetization ensures that the right user, sees the right ad, at the right time.  These are the moments we monetize in your application:
+
+* [__Reward__](doc/make-money/reward-moment): congratulatory brand sponsored reward after a moment of achievement 
+* [__Rescue__](doc/make-money/rescue-moment): saves a user in a moment of need through brand sponsored value exchange ads
+* [__Break__](doc/make-money/break-moment): provides custom brand messages during natural pause moment in app action
+
+You must get approval from a member of the Tap for Tap support team to activate moments based interstials into your application. This ensures moments are properly placed for top performace. 
```

Comment:
How do you get approval? Is there an email address you can send your request to?